src/pfe/iterative-dev/idc-java/IDC/src/org/eclipse/codewind/iterdev/IDCContext.java

```diff
@@ -149,6 +151,14 @@ public IDCContext(String rootPassword, String localWorkspaceOrigin, String conta
 		else {
 			this.imageCommand = "docker";
 		}
+
+		if (this.isK8s) {
+			this.imagesFormatString = "\"{{.Name}}\"";
```

Comment:
Can you please add a comment here explaining that it's different in K8s because buildah's list format has changed from {{.Repository}} to {{.Name}}